added modified hf codes

rabeehkarimimahabadi opened the pull request.

georgedahl commented on seq2seq/third_party/models/__init__.py:
```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
```
Can you reply to this comment with the link where you copied this file from? Or did you write it? If you wrote it, remove the HF copyright notice.

rabeehkarimimahabadi replied:
thanks George, removed.

georgedahl commented on seq2seq/third_party/models/configuration_t5.py:
```diff
@@ -14,8 +14,8 @@
 # limitations under the License.
```
This comment is about adding text to line 3.

Can you add a comment as the doc I sent you discusses (indicating this file has been modified etc.)?

rabeehkarimimahabadi replied:
thanks, done

georgedahl replied:
Great, looks good. Please add a new line back in for clarify before the License part.

georgedahl commented on seq2seq/third_party/models/modeling_outputs.py:
```diff
@@ -0,0 +1,126 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
```
Is this a brand new file you wrote entirely from scratch, as it appears to be? If so, since it isn't an __init__.py file, please move it out of third_party and correct the copyright notice.

If it isn't, can you link me to the original version of the file?

rabeehkarimimahabadi replied:
Please find the original file here: https://github.com/huggingface/transformers/blob/v3.5.1/src/transformers/modeling_outputs.py 

I only modified 'BaseModelOutputWithPastAndCrossAttentions' and 'Seq2SeqLMOutput'

georgedahl commented on seq2seq/third_party/models/modeling_t5.py:
```diff
@@ -14,409 +14,63 @@
 # limitations under the License.
```
This file has been modified from its original, so text will need to be inserted at line 3 as described in the doc I sent you.

rabeehkarimimahabadi replied:
done, thanks

georgedahl commented on seq2seq/third_party/trainers/t5_trainer.py:
```diff
@@ -1,7 +1,24 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
+#
```
Since this looks like a heavily modified version of the original HuggingFace file, we need to indicate that it has been modified here, as described in the doc I sent you.

rabeehkarimimahabadi replied:
thanks, done

georgedahl commented on seq2seq/third_party/trainers/trainer.py:
```diff
@@ -28,17 +28,16 @@
 import warnings
```
This comment refers to line 3. Similarly, this is a modified version of the original HuggingFace file so we need to indicate it has been changed as described in the doc I sent you.

rabeehkarimimahabadi replied:
thanks, done

georgedahl commented on seq2seq/third_party/utils/sentence_splitter.py:
```diff
@@ -1,3 +1,16 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
```
This file looks unmodified other than adding license information. Maybe in this case we should leave it unchanged and only clarify the headers if we have to modify it ourselves later.

rabeehkarimimahabadi replied:
so you mean to remove license? Jamie once said we need to have a license fle on top of every file, what do you think? thanks

georgedahl replied:
I am not 100% sure. I guess this is fine.

georgedahl commented on seq2seq/third_party/utils/utils.py:
```diff
@@ -1,3 +1,16 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
+#
```
it looks like we have made a very minor modification to this file. Is the modification necessary or will it work if we revert it? If it isn't necessary, please revert it.

If it is necessary, we should add the usual comment about how this file has been modified (as described in the doc).

rabeehkarimimahabadi replied:
this is not modified, more than formatting and changing the imports which needs to be done, handled it as category 3.

georgedahl commented on seq2seq/third_party/trainers/__init__.py:
```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
```
Did you write this file or copy it? If you wrote it, correct the copyright line. If you copied it (and potentially modified it) can you link me to where you copied it from? 

__init__.py files from the HF project might in some cases have to be totally replaced by us. If they are completely replaced and nothing was copied, we can treat them like any other code file that we write from scratch except, because they are special and need to be located in a specific place, we will allow them to go in third_party.

rabeehkarimimahabadi replied:
I wrote it, thanks, done

georgedahl commented on seq2seq/third_party/utils/__init__.py:
```diff
@@ -0,0 +1,11 @@
+from .utils import (
```
I assume you wrote this file from scratch? Can you just confirm that? If that is true, it is fine, but if not please link me to the original version.

rabeehkarimimahabadi replied:
I wrote it

rabeehkarimimahabadi commented:
Hi George,
Thanks for the comments, I addressed them and updated the pool request.
Best
Rabeeh

On Mon, Dec 28, 2020 at 8:57 AM George Dahl <notifications@github.com>
wrote:

> *@georgedahl* requested changes on this pull request.
> ------------------------------
>
> In seq2seq/third_party/models/__init__.py
> <https://github.com/google-research/ruse/pull/15#discussion_r549259887>:
>
> > @@ -0,0 +1,15 @@
> +# Copyright 2020 The HuggingFace Team. All rights reserved.
>
> Can you reply to this comment with the link where you copied this file
> from? Or did you write it? If you wrote it, remove the HF copyright notice.
> ------------------------------
>
> In seq2seq/third_party/models/configuration_t5.py
> <https://github.com/google-research/ruse/pull/15#discussion_r549261815>:
>
> > @@ -14,8 +14,8 @@
>  # limitations under the License.
>
> This comment is about adding text to line 3.
>
> Can you add a comment as the doc I sent you discusses (indicating this
> file has been modified etc.)?
> ------------------------------
>
> In seq2seq/third_party/models/modeling_outputs.py
> <https://github.com/google-research/ruse/pull/15#discussion_r549262428>:
>
> > @@ -0,0 +1,126 @@
> +# Copyright 2020 The HuggingFace Team. All rights reserved.
>
> Is this a brand new file you wrote entirely from scratch, as it appears to
> be? If so, since it isn't an *init*.py file, please move it out of
> third_party and correct the copyright notice.
>
> If it isn't, can you link me to the original version of the file?
> ------------------------------
>
> In seq2seq/third_party/models/modeling_t5.py
> <https://github.com/google-research/ruse/pull/15#discussion_r549262818>:
>
> > @@ -14,409 +14,63 @@
>  # limitations under the License.
>
> This file has been modified from its original, so text will need to be
> inserted at line 3 as described in the doc I sent you.
> ------------------------------
>
> In seq2seq/third_party/trainers/t5_trainer.py
> <https://github.com/google-research/ruse/pull/15#discussion_r549263851>:
>
> > @@ -1,7 +1,24 @@
> +# Copyright 2020 The HuggingFace Team. All rights reserved.
> +#
>
> Since this looks like a heavily modified version of the original
> HuggingFace file, we need to indicate that it has been modified here, as
> described in the doc I sent you.
> ------------------------------
>
> In seq2seq/third_party/trainers/trainer.py
> <https://github.com/google-research/ruse/pull/15#discussion_r549264768>:
>
> > @@ -28,17 +28,16 @@
>  import warnings
>
> This comment refers to line 3. Similarly, this is a modified version of
> the original HuggingFace file so we need to indicate it has been changed as
> described in the doc I sent you.
> ------------------------------
>
> In seq2seq/third_party/utils/sentence_splitter.py
> <https://github.com/google-research/ruse/pull/15#discussion_r549265784>:
>
> > @@ -1,3 +1,16 @@
> +# Copyright 2020 The HuggingFace Team. All rights reserved.
>
> This file looks unmodified other than adding license information. Maybe in
> this case we should leave it unchanged and only clarify the headers if we
> have to modify it ourselves later.
> ------------------------------
>
> In seq2seq/third_party/utils/utils.py
> <https://github.com/google-research/ruse/pull/15#discussion_r549266643>:
>
> > @@ -1,3 +1,16 @@
> +# Copyright 2020 The HuggingFace Team. All rights reserved.
> +#
>
> it looks like we have made a very minor modification to this file. Is the
> modification necessary or will it work if we revert it? If it isn't
> necessary, please revert it.
>
> If it is necessary, we should add the usual comment about how this file
> has been modified (as described in the doc).
> ------------------------------
>
> In seq2seq/third_party/trainers/__init__.py
> <https://github.com/google-research/ruse/pull/15#discussion_r549267735>:
>
> > @@ -0,0 +1,14 @@
> +# Copyright 2020 The HuggingFace Team. All rights reserved.
>
> Did you write this file or copy it? If you wrote it, correct the copyright
> line. If you copied it (and potentially modified it) can you link me to
> where you copied it from?
>
> *init*.py files from the HF project might in some cases have to be
> totally replaced by us. If they are completely replaced and nothing was
> copied, we can treat them like any other code file that we write from
> scratch except, because they are special and need to be located in a
> specific place, we will allow them to go in third_party.
> ------------------------------
>
> In seq2seq/third_party/utils/__init__.py
> <https://github.com/google-research/ruse/pull/15#discussion_r549268056>:
>
> > @@ -0,0 +1,11 @@
> +from .utils import (
>
> I assume you wrote this file from scratch? Can you just confirm that? If
> that is true, it is fine, but if not please link me to the original version.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/google-research/ruse/pull/15#pullrequestreview-559043651>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARPXHH7BPZL3QE7IG3NU6VTSXBCAZANCNFSM4VHQQLKQ>
> .
>

georgedahl commented on seq2seq/third_party/__init__.py:
```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Google LLC
```
This looks like this file
https://github.com/huggingface/transformers/blob/v3.5.1/examples/seq2seq/__init__.py
So unless you wrote it independently, we should mark it as copied from HF and treat it as a file in category 2 file from the doc.
